scripts/build.sh

```diff
@@ -19,4 +19,9 @@ export CGO_ENABLED=0
 
 # Build binary
 cd $GOPATH/src/github.com/raintank/metrictank
-go build -ldflags "-X main.GitHash=$GITVERSION" -o $BUILDDIR/metrictank
+if [ "$1" == "race" ]
+then
+  CGO_ENABLED=1 go build -race -ldflags "-X main.GitHash=$GITVERSION" -o $BUILDDIR/metrictank.race
```

Comment:
isn't it impractical to give the binary a different name since everything assumes the binary is named `metrictank` (e.g. the docker stack). how would you run metrictank.race?